bin/hoc-2015-survey/test-params-organizers.csv

```diff
@@ -0,0 +1,5 @@
+email,name,international,domestic,no_posters
```

Comment:
This is handled by the csv export code.

On Fri, Dec 18, 2015 at 2:44 PM, Mehal Shah <notifications@github.com>
wrote:

> In bin/hoc-2015-survey/test-params-organizers.csv
> <https://github.com/code-dot-org/code-dot-org/pull/6144#discussion_r48078390>
> :
>
> > @@ -0,0 +1,5 @@
> > +email,name,international,domestic,no_posters
>
> How do we handle someone with a comma in their name? Will it get escaped
> properly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6144/files#r48078390>.
>